tools/eol_cve/eol-cve.csv

```diff
@@ -0,0 +1,120 @@
+cve,vulnerable,patched,affectedEOL
+CVE-2016-9551,^7.1.0,^7.2.0,"4.x || 5.x || 6.x"
```

Comment:
This CVE has been patched to v7.2.0 and it was released in 2016 (https://github.com/nodejs/node/releases/tag/v7.2.0) and v6.x has been EOL only in 2019-04-30 https://github.com/nodejs/release?tab=readme-ov-file#end-of-life-releases, so it means v6.x is not affected by this CVE.
 

<br class="Apple-interchange-newline">

Reply:
We can take the date of the release of the patched version, and correlate it with the date of EOL for each missing version. Getting this data is not easy